app/editor/src/index.scss

```diff
@@ -103,6 +103,10 @@ h4 {
             cursor: pointer;
         }
     }
+    &.chk.allow-no-file {
```

Comment:
Wondering if this styling should be moved to either the `ContentForm` or `Check` styled component? @Fosol

Reply:
Yes move to the ContentForm styled component, as it is only relevant to that component.

Reply:
Updated, thanks